src/content/docs/en/guides/upgrade-to/v5.mdx

```diff
@@ -753,6 +758,10 @@ export const onRequest = defineMiddleware(async (context, next) => {
 })
 ```
 
+:::note[Cookie path configuration]
+The `path: '/'` option is critical when using `trailingSlash: "ignore"` (the default with `build.format: "directory"`). Without an explicit path, the cookie defaults to the request's directory path, which can cause cookies to be inaccessible after redirects between `/route` and `/route/`. Setting `path: '/'` ensures the cookie is available regardless of trailing slashes.
+:::
```

Comment:
```suggestion
```
Discussed with the team and we're going to leave the note out, but keep the extra "best practice" in the example as being more helpful!